build-output-api/on-demand-isr-revalidation/.vercel/output/functions/revalidate.func/index.js

```diff
@@ -0,0 +1,37 @@
+const https = require('https');
+
+// The bypass token can be a randomly generated string of at least 32 characters.
+// This is meant to be *private* - DO NOT expose this value on the client-side.
+const bypassToken = '87734ad8259d67c3c11747d3e4e112d0'
+
+module.exports = (req, res) => {
+  const proto = req.headers['x-forwarded-proto'];
```

Comment:
This should always be `https` on Vercel so I don't think we need to detect this from the header here. 
```suggestion
```